src/main/java/club/issizler/okyanus/api/Mod.java

```diff
@@ -33,16 +33,16 @@ public Server getServer() {
      * @param e The event handler to register
      */
     public void registerEvent(EventHandler e) {
-        getServer().registerEvent(e);
+        getServer().getEventRegistry().register(e);
     }
 
     /**
-     * Registers a {@link CommandBuilder}
+     * Registers a {@link ICommand}
      *
-     * @param builder The command to register
+     * @param command The command to register
      */
-    public void registerCommand(CommandBuilder builder) {
-        getServer().registerCommand(builder);
+    public void registerCommand(ICommand command) {
```

Comment:
How would registering the old builder-based commands work if you are just using a single method for registration. Shouldn't you have a deprecated compatibility method for that?

ps: You might want to test your changes against a mod like https://github.com/okyanus-mods/OKAuth that contains old builder-based commands.

Reply:
yes i didn't add it, i'll